### ISSUE TITLE
Japanese Fonts Too Small / Halo (Stroke) too small / Shields misrendered

### ISSUE THREAD

**Issue by javbw** [gravitystorm/openstreetmap-carto#294]
A lot of the labels in Japanese are using a light stroke font instead of a bolded font when the English is rendered bold. 

Train stations, amusement parks, motorways, and other important places have hard names to read in non-roman characters because they are rendered smaller than they should be for the kanji.

Examples:

Train Stations http://www.openstreetmap.org/#map=17/36.41395/139.33339
The Kanji for Nishikiryu is difficult to read.

Parks http://www.openstreetmap.org/#map=18/35.73297/139.74664
(The Rikugi-en Garden's complicated kanji is unreadable, for example) 

Tokyo Disneyland http://www.openstreetmap.org/#map=15/35.6293/139.8818
Tokyo Disneyland's label (東京ディズニーランド) is impossible to see, let alone read. (thin stroke, no contrast, tiny size) The labels on the parking isles? totally visible.  (A whole 'nother issue issue of not rendering labels on parking isles at lower zoom levels)  

Motorway labels http://www.openstreetmap.org/#map=15/36.3551/139.1981

The shield labels for the motorways are rendered completely wrong, overlapping the kanji.  The Kanji font is very feint.  Contrast this to the trunk label (17) right above it. 
Also the exit labels are not very readable either.  Usually you need these labels when going to a new place, so it is difficult to guess complicated kanji or read faint ones

 - - - 

Most of these issues can be remedied by the choice of a better font that includes bolded characters for Japanese / Chinese. I assume you are using the same font for these  features in all regions, but that font doesn't seem to have bold weight 6 characters (W6).  The font choice is also a serif font (you can see the strokes in the characters) for some features, when that is unneeded and makes things very hard to read at the small level they are rendered, whereas a san-serif font (no strokes) which is used in other places, is perfect for this Job. 

TL;DR : Japanese characters are faint / garbled / rendered incorrectly in some situations. Update the font to a san-serif with bold characters available, or at lease use only san-serif fonts for non-roaman languages, or increase the font size.

**Comment by math1985** [comment]
Did anybody test whether Mapnik 3 improves the situation?

**Comment by dink-straycat** [comment]
That's my tileserver's output(CentOS7, Apache + mod_tile, mapnik v3.0.9).
Right-side is OSM (Maybe mapnik2?).

Nishikiryu:
![mapnik3-001](https://cloud.githubusercontent.com/assets/704841/12667625/86b5e3dc-c691-11e5-9995-c8da5959dcf6.png)

The Rikugi-en Garden:
![mapnik3-002](https://cloud.githubusercontent.com/assets/704841/12667626/8b3c5fb2-c691-11e5-9c6d-63d251f7c5c3.png)

Tokyo Disneyland:
![mapnik3-003](https://cloud.githubusercontent.com/assets/704841/12667627/8e006298-c691-11e5-8927-bd8abe86d260.png)

North Kanto Expressway:
![mapnik3-004](https://cloud.githubusercontent.com/assets/704841/12667630/90c9413e-c691-11e5-9d22-71d785384061.png)


Maybe, there is no problem about this issue,
but Japanese label's word-wrap seems to be worse on mapnik3.

**Comment by sommerluk** [comment]
As far as I see, the following things have been discussed here:

- Many asian scripts are a bitmap font and not a vector font → SOLVED
- Many asian scripts have no bold variant → OPEN (see #2240 for Korean script)
- Many asian scripts  need a bigger font size than latin text for being legible → OPEN (see #2265)
- Many asian scripts are barely legible on road shields→ SOLVED
- Various proposals for rendering features at lower or higher zoom levels: https://github.com/gravitystorm/openstreetmap-carto/issues/294#issuecomment-31897583 → Status???

I’m not sure about the proposed feature/zoom level changes. But for the rest, I propose to close this issue and continue the work in the specialized issues.
